src/main/java/Common/CommonFunctions.java

```diff
@@ -0,0 +1,19 @@
+package Common;
+
+public class CommonFunctions {
```

Comment:
Could maybe consider changing the class name to Ui/Printer so that it is clear that the class handles the printing of outputs